jsonschema-validation.xml

```diff
@@ -418,6 +435,12 @@
                         result is a boolean "true" and the instance array length is less than or
                         equal to the "maxContains" value.
                     </t>
+                    <t>
+                      If annotations are not being collected, the validation value may be determined
```

Comment:
This paragraph is simply describing the alternative algorithm to use that doesn't involve annotations. MAY in the RFC sense isn't correct here as alternative behaviour is not permitted -- it's only the implementation can vary.  How about "can"?

Reply:
I was just going by this language from the `additional*` applicators:

>    Implementations MAY choose to implement or optimize this keyword in
   another way that produces the same effect, such as by directly
   checking the names in "properties" and the patterns in
   "patternProperties" against the instance property set.
   Implementations that do not support annotation collection MUST do so.

As long as they end up consistent I'm probably fine with whatever.  If making it consistent gets bogged down, I'll live with a little inconsistency.  It is probably better to use "can" than lower-case "may", although that is definitely not an absolute rule.

Reply:
Sounds like a reaonsable change. Let's go with "can".

Reply:
I'm cutting out this entire part of the revision, since (as per other PRs recently submitted) trying to define a parallel implementation using annotations is not going to be identicaly, so we shouldn't suggest it at all.